extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -221,60 +233,59 @@ private function getParentWallMessageMock( array $params ) {
 			] )
 			->disableOriginalConstructor()
 			->getMock();
+		$parentWallMessageCounter = 0;
 
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'load' );
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'getTitle' )
-			->willReturn( $parentTitle );
-		$parentWm->expects( $this->any() )
+			->willReturn( $parentTitleMock );
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'getMetaTitle' )
 			->willReturn( $params['parentWallMessage']['metaTitle'] );
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
+			->method( 'getUser' )
+			->willReturn( $parentUserMock );
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'getId' )
 			->willReturn( $params['parentWallMessage']['id'] );
-		$parentWm->expects( $this->once() )
-			->method( 'getUser' )
-			->willReturn( $parentUser );
 
-		return $parentWm;
+		return $parentWallMessageMock;
 	}
 
-	private function getParentTitleMock( array $params ) {
-		$parentTitle = $this->getMockBuilder( 'Title' )
-			->setMethods( [
-				'getDBkey',
-			] )
-			->disableOriginalConstructor()
-			->getMock();
+	private function getParentTitleMock( array $params ): PHPUnit_Framework_MockObject_MockObject {
+		$parentTitleMock = $this->getMock( Title::class, [ 'getDBkey' ] );
 
-		$parentTitle->expects( $this->once() )
+		$parentTitleMock->expects( $this->once() )
 			->method( 'getDBkey' )
 			->willReturn( $params['parentTitle']['dbKey'] );
 
-		return $parentTitle;
+		return $parentTitleMock;
 	}
 
-	private function getParentUserMock( array $params ) {
-		$parentTitle = $this->getMockBuilder( 'User' )
-			->setMethods( [
-				'getId',
-				'getName',
-			] )
+	private function getParentUserMock( array $params ): PHPUnit_Framework_MockObject_MockObject {
+		$parentUserMock = $this->getMockBuilder( User::class )
+			->setMethods( [ 'getId', 'getName' ] )
 			->disableOriginalConstructor()
 			->getMock();
 
-		$parentTitle->expects( $this->once() )
-			->method( 'getId' )
-			->willReturn( $params['parentUser']['id'] );
-		$parentTitle->expects( $this->once() )
+		$parentUserMock->expects( $this->once() )
 			->method( 'getName' )
 			->willReturn( $params['parentUser']['name'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "parentUser" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)